producer/src/main/java/by/shilay/app/producer/service/api/EventService.java

```diff
@@ -0,0 +1,8 @@
+package by.shilay.app.producer.service.api;
+
+import by.shilay.app.producer.model.Event;
+
+public interface EventService {
+
+    void save(Event event);
+}
```

Comment:
Excess interface, remove this because you don't use Remote Procedure Connection.

Reply:
got it:)